configs/sysbench_memory_example.yaml

```diff
@@ -1,7 +1,8 @@
 threads: 5
 events: 0
 time: 15
-memory-block-size: '2KiB'
-memory-total-size: '100GiB'
-memory-scope: 'local'
-memory-oper: 'write'
\ No newline at end of file
+memoryblocksize: '2KiB'
+memorytotalsize: '100GiB'
+memoryscope: 'local'
+memoryoperation: 'write'
```

Comment:
These keys were all changed with PR #12 to use the [hyphenated format](https://github.com/arcalot/arcaflow-plugin-sysbench/blob/611ec16229d66ab524bc8a07338194ab58e24c9d/arcaflow_plugin_sysbench/sysbench_schema.py#L190) to match exactly with the existing sysbench command input in order to reduce confusion between the plugin and the tool.